pkg/scheduler/objects/application_state.go

```diff
@@ -143,6 +143,7 @@ func NewAppState() *fsm.FSM {
 			},
 			fmt.Sprintf("enter_%s", Starting.String()): func(event *fsm.Event) {
 				app := event.Args[0].(*Application) //nolint:errcheck
+				app.queue.incRunningApps()
```

Comment:
This is the correct point (see the [usage tracking doc](https://docs.google.com/document/d/19FYOL1Uw7tSf74qyuVN40Fv-GdWd2KrBcHJpUQLTBZ4/edit#heading=h.481iug6rohb3)) but we need to make sure the metrics are updated in the same place as the queue itself.

Reply:
Updated in https://github.com/apache/yunikorn-core/pull/435

Reply:
Updated